libtiledbsoma/src/soma/managed_query.h

```diff
@@ -460,6 +460,14 @@ class ManagedQuery {
         Enumeration enmr,
         ArraySchemaEvolution& se);
 
+    bool _extend_enumeration(
```

Comment:
Optional: Even though this is an internal function, a short doxygen summary can be helpful. Something like "Modifies the the array schema to contain the extended enumeration." Same for other new functions in the file.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/3815/commits/532958e69f4763fce56b88ab1c7d9159ce78b921